### ISSUE TITLE
100% CPU

### ISSUE THREAD

**Issue by collinpeters** [a85/POSTMan-Chrome-Extension#3167]
## App Details:

```
Postman for Linux
Version 5.0.0
linux 4.8.0-54-generic / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: recently
 2. Expected behaviour: not 100% CPU :)
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app): Perhaps related, but I cannot actually click the View menu at the moment. The rest of Postman works fine... but the menu is not responsive. I will play with this a bit because I am using KDE with the Global Menu applet, which could definitely be the cause of it.
 4. Screenshots (if applicable)

**Comment by kamalaknn** [comment]
Hey @collinpeters 

Thanks for sending your feedback.

We are working on some performance improvements around memory and CPU usage.

Could you share more information on when exactly you see this high CPU usage? Is it when you perform a particular action? Also, are you on a native linux machine or a virtual machine?

Thanks.

**Comment by ChALkeR** [comment]
If this app is built with `Electron` (which I guess from the syptoms and a quick Google search on the app name), then this is caused by the upstream Electron issue: https://github.com/electron/electron/issues/8455.

**Comment by sdnts** [comment]
@collinpeters @ChALkeR Are you guys still seeing this?

**Comment by ChALkeR** [comment]
@madebysid I am not a user of Postman.

But the issue was resolved on Electron side in v1.7.6/v1.6.12, so if your Electron version is newer than that, this should be fixed. Those are both pre-releases, though.

If you don't bundle Electron and use distribution-provided one in Postman, then I guess this could be caused as upstream, and if you bundle Electron — with the bundled Electron version bump to a version that includes that patch.
